packages/framework/src/Component/Elasticsearch/Debug/ElasticsearchDebugStack.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Elasticsearch\Debug;
+
+class ElasticsearchDebugStack
```

Comment:
`stack` is a data structure with different properties than this data structure.

Please rename this class to match the responsibility. The best name I can come with is `ElasticsearchRequestCollection`